src/Framework/App.Runtime/src/aspnetcore-runtime-composite.proj

```diff
@@ -0,0 +1,66 @@
+<Project Sdk="Microsoft.Build.Traversal">
+  <PropertyGroup>
```

Comment:
What's the purpose of this proj?

Reply:
This project builds an archive containing the .NET runtime and the ASP.NET Core runtime with the Composite ReadyToRun image (which is used in some of our docker images).